xyz-hub-service/src/main/java/com/here/xyz/hub/config/SpaceConfigClient.java

```diff
@@ -173,6 +173,8 @@ public Future<List<Space>> getSpacesForOwner(Marker marker, String ownerId) {
   protected abstract Future<List<Space>> getSelectedSpaces(Marker marker, SpaceAuthorizationCondition authorizedCondition,
       SpaceSelectionCondition selectedCondition, PropertiesQuery propsQuery);
 
+  public abstract Future<List<Space>> getSpacesFromParent(Marker marker, String parentSpaceId);
```

Comment:
On other places, we're calling it the "super" space already. In order to not introduce any confusion here, it could be renamed to `#getSpacesFromSuper(...)` .... or even better: `#getExtendingSpaces(Marker marker, String superSpaceId)`

Reply:
Extending and extension are confusing terms, will use super.